<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSettingsDialog</class>
 <widget class="QDialog" name="ColorSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Settings</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout" stretch="0,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="XmlUi::GradientWidget" name="promptWidget" native="true">
     <layout class="QHBoxLayout" name="promptLayout" stretch="0,1">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="promptPixmap"/>
      </item>
      <item>
       <widget class="QLabel" name="promptLabel">
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <widget class="QGroupBox" name="colorGroupBox">
        <property name="title">
         <string>Default Color</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="modeLabel">
           <property name="text">
            <string>Color &amp;mode:</string>
           </property>
           <property name="buddy">
            <cstring>modeComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="modeComboBox">
           <item>
            <property name="text">
             <string>Single Color</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dual Colors</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>253</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="primaryLabel">
           <property name="text">
            <string>&amp;Primary color:</string>
           </property>
           <property name="buddy">
            <cstring>primaryButton</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ColorButton" name="primaryButton">
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="secondaryLabel">
           <property name="text">
            <string>&amp;Secondary color:</string>
           </property>
           <property name="buddy">
            <cstring>secondaryButton</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="ColorButton" name="secondaryButton">
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="backgroundGroupBox">
        <property name="title">
         <string>Scene Background</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="bgModeLabel">
           <property name="text">
            <string>Bac&amp;kground mode:</string>
           </property>
           <property name="buddy">
            <cstring>bgModeComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="bgModeComboBox">
           <item>
            <property name="text">
             <string>Solid</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gradient</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>245</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="topLabel">
           <property name="text">
            <string>&amp;Top color:</string>
           </property>
           <property name="buddy">
            <cstring>topButton</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ColorButton" name="topButton">
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="bottomLabel">
           <property name="text">
            <string>&amp;Bottom color:</string>
           </property>
           <property name="buddy">
            <cstring>bottomButton</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="ColorButton" name="bottomButton">
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XmlUi::GradientWidget</class>
   <extends>QWidget</extends>
   <header>xmlui/gradientwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorButton</class>
   <extends>QToolButton</extends>
   <header>widgets/colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>modeComboBox</tabstop>
  <tabstop>primaryButton</tabstop>
  <tabstop>secondaryButton</tabstop>
  <tabstop>bgModeComboBox</tabstop>
  <tabstop>topButton</tabstop>
  <tabstop>bottomButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColorSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColorSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
